pkgs/web_socket/lib/src/web_socket.dart

```diff
@@ -112,6 +115,9 @@ class WebSocketConnectionClosed extends WebSocketException {
 ///   socket.sendText('Hello Dart WebSockets! 🎉');
 /// }
 abstract interface class WebSocket {
+  static Future<WebSocket> connect(Uri url, {Iterable<String>? protocols}) =>
```

Comment:
Can you add a doc comment? Can you describe the behavior of the `protocols` argument, and consider including the information that it is ignored on the web?

Reply:
I have a follow-up PR that enables the `protocol` functionality and adds comments: https://github.com/dart-lang/http/pull/1150

I wanted to add a cross-platform connect in this PR but realized that I had a non-function `protocol` argument in the tests but not in connect. So I wanted to fix that without implementing the functionality in the same PR - it felt like too much for one commit.